dbt_schema_builder/builder.py

```diff
@@ -617,3 +713,16 @@ def run(self):
                 logger.info('\n')
                 logger.info('------- {} -------'.format(app_name))
                 self.builder.build_app(app_name, app_config)
+                
+    def run_no_pii(self):
```

Comment:
You may want to reduce your code duplication a little bit, don't be afraid to change the method signature in a non-breaking way and see how far down the code you can push something. For example you could redefine the above method signature `def run(self):` into something like `def run(self, nopii=False):`

Reply:
@dnuzz Appreciate it, have changed solution accordingly